mmo.go

```diff
@@ -419,3 +423,15 @@ func debug(c *cli.Context, message string) {
 		"command": c.Command.Name,
 	}).Debugln(message)
 }
+
+func getPrefix() (string, error) {
```

Comment:
Please add test. Also avoid the side-effect of os.Getwd by passing the argument into the function and using os.Getwd directly within the action